docs/MapColonies/Raster/services/PolygonParts/README.md

```diff
@@ -55,6 +85,8 @@ A{Get Auth Token} -->|token| B[GetCapabilities]
     end
 ```
 
+### List  WFS capabilities and find available FeaturesTypes
+
 To list all the available feature types use the `GetCapabilities` operation and look for the `FeatureTypeList` section.
```

Comment:
Please add an example for the `GetCapabilities` request.

Reply:
the example response xml is too long, that's why I mentioned and show the relevant scope from the response body in the example